packages/ui5-middleware-livereload/lib/livereload.js

```diff
@@ -63,8 +63,19 @@ module.exports = async ({ resources, options }) => {
             log.info("Livereload server started!");
         }
     );
-    debug ? log.info(`Livereload connecting to port ${port} for path ${watchPath}`) : null;
-    livereloadServer.watch(path.join(process.cwd(), watchPath));
+    
+	if (Array.isArray(watchPath)) {
+		let watchPaths = [];
+		for (let i = 0; i < watchPath.length; i++) {
+			watchPaths.push(path.join(process.cwd(), watchPath[i]));
+			debug ? log.info(`Livereload connecting to port ${port} for path ${watchPath[i]}`) : null;
```

Comment:
I would move the debug line to before the watch command and just log all "paths" as array.

```
debug ? log.info(`Livereload connecting to port ${port} for paths ${watchPaths}`) : null;
```

Reply:
OK, I will make another commit.